osf/utils/migrations.py

```diff
@@ -197,9 +197,10 @@ def create_schema_block(state, schema_id, block_type, display_text='', required=
     """
     For mapping schemas to schema blocks: creates a given block from the specified parameters
     """
-    RegistrationSchemaBlock = state.get_model('osf', 'registrationschemablock')
+    state = state or apps
+    schema_block_model = state.get_model('osf', 'registrationschemablock')
```

Comment:
Drive-by: A little bit of a strange situation because we getting the model in different way, but model definitions always follow camel case.